### DISCUSSION TITLE
Can u provide a simple verion that can be managed by systemd ?

### DISCUSSION THREAD

**Discussion by yesiamchinese** [enfein/mieru#170] [Ideas]
Would you please release a simple mieru verion that can be managed by systemd easily ?

A simple verion only support a few simple command, such as :
mieru -c /etc/mieru/config.json # run and start with config file /etc/meru/config.json
mieru -t /etc/mieru/config.json # test if the config file is valide
mieru -v # show version

If so, we can setup more than one mieru socks service for mihomo:
mieru -c /etc/mieru/server1.json # provide service in socks 1080
mieru -c /etc/mieru/server2.json # provide service in socks 1081
......

Then, All mita server node can be easily managed by mihomo GUI such like METACUBE or yacd.

You can see naiveproxy, hysteria(2) etc is very simple.

**Comment by enfein** [comment]
See an example systemd service at https://github.com/enfein/mieru/blob/main/configs/examples/mieru.service

The environment variable is explained at https://github.com/enfein/mieru/blob/main/docs/operation.md#environment-variables

**Reply by yesiamchinese** [reply]
After I run "systemctl start mieru", I still need to run "mieru run" again before it can work normally.
Is it possible that after I run "systemctl start mieru" (and it automatically runs "mieru run"), it can then work directly without needing to run "mieru ru" again?

**Reply by enfein** [reply]
It shouldn't. According to https://github.com/enfein/mieru/blob/7f828fca55485d26f7561264d7afa86d01709103/configs/examples/mieru.service#L11 `mieru run` is executed when you run `systemctl start mieru`.
